app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -42,6 +42,10 @@ def sp_name
         service_provider ? service_provider.friendly_name : APP_NAME
       end
 
+      def service_provider_homepage_url
+        sp_return_url_resolver.homepage_url if service_provider
+      end
+
```

Comment:
It'd be good to add both `service_provider_homepage_url` and `sp_return_url_resolver` to [the presenter spec](https://github.com/18F/identity-idp/blob/main/spec/presenters/idv/in_person/verification_results_email_presenter_spec.rb).

Reply:
I've added specs for the public interface, `service_provider_homepage_ur`. I tend to think the private methods are implementation details that should be tested through public interfaces

Reply:
See [3583311](https://github.com/18F/identity-idp/pull/7526/commits/3583311e9439f38408b3d7fca6837eced11a0219)